instance/instance.go

```diff
@@ -109,6 +109,71 @@ func (s *Store) Add(w http.ResponseWriter, r *http.Request) {
 	log.Debug("add instance", log.Data{"instance": instance})
 }
 
+// UpdateDimension updates label and/or description for a specific dimension within an instance
+func (s *Store) UpdateDimension(w http.ResponseWriter, r *http.Request) {
```

Comment:
are there unit tests to cover this function? if not we should add some (not fun but it's necessary)